assets/css/style.css

```diff
@@ -0,0 +1,8 @@
+.color-cell {
+    width: 100px;
+    height: 100px;
+}
```

Comment:
if this is meant for image scale down, please choose either `width` or `height` but not both. any image with aspect ration different than 1:1 will distort